app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
+
+
+def create_person_list(people: list[dict]) -> list:
+    for person in people:
+        Person(person["name"], person["age"])
+    for person in people:
+        current_person = Person.people[person["name"]]
+        if "wife" in person and person["wife"]:
```

Comment:
You can simplify this row with checking only 'if person.get("wife"):' and you will have those two checks in one.